### DISCUSSION TITLE
[Feature] Mapbox support as a frontend map provider

### DISCUSSION THREAD

**Discussion by freak12techno** [immich-app/immich#3435] [Feature Request]
### The feature

It'd be nice if we'd be able to have Mapbox as an alternative to Leaflet, as it's working way more smooth.
Ideal would be if a user would be able to choose a map provider and use Leaflet as a default (as Mapbox requires a key for all the requests).
Do you think it is possible?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4294 will switch the map library to Maplibre, which should also support using mapbox as a provider.

**Comment by freak12techno** [comment]
@bo0tzz would it allow for vector tiles instead of scalar ones? it should make the UI more smooth

**Reply by bo0tzz** [reply]
Yes

**Comment by bo0tzz** [comment]
Closed by #4294
